src/sentry/api/endpoints/organization_sessions.py

```diff
@@ -81,7 +80,7 @@ def handle_query_errors(self):
             # TODO: this context manager should be decoupled from `OrganizationEventsEndpointBase`?
             with super().handle_query_errors():
                 yield
-        except (InvalidField, InvalidParams, NoProjects, InvalidSearchQuery) as error:
+        except (InvalidField, InvalidParams, NoProjects) as error:
```

Comment:
This was removed because it is redundant as this is already handled in `super().handle_query_errors()`